libpod/stats.go

```diff
@@ -0,0 +1,126 @@
+package libpod
+
+import (
+	"path/filepath"
+	"strings"
+	"syscall"
+	"time"
+
+	"github.com/containerd/cgroups"
+	"github.com/opencontainers/runc/libcontainer"
+	"github.com/pkg/errors"
+)
+
+// ContainerStats contains the statistics information for a running container
+type ContainerStats struct {
+	Container   string
+	CPU         float64
+	cpuNano     uint64
```

Comment:
This and systemNano are both not exported, so functions outside of libpod can't see them - is this intentional?

Reply:
Yeah I think so ... remember this code was initially done by someone else.

Reply:
I think we should just capitalize them... I don't think it will break anything, and I see nothing wrong with letting people see cpuNano

Reply:
ok fixed